packaging/systemd/cri-docker.socket

```diff
@@ -3,7 +3,7 @@ Description=CRI Docker Socket for the API
 PartOf=cri-docker.service
 
 [Socket]
-ListenStream=/var/run/cri-docker.sock
+ListenStream=$t/cri-docker.sock
```

Comment:
This is incorrect, it's supposed to be `%t` not `$t`.

Reply:
See #9

Reply:
Right, it's using `%t`.